.editorconfig

```diff
@@ -16,6 +16,11 @@ ij_smart_tabs = false
 ij_visual_guides = none
 ij_wrap_on_typing = false
 
+# Specific settings for YAML files
+[{*.yml,*.yaml}]
+indent_style = space
+indent_size = 2
+
```

Comment:
🤔 tabs vs spaces? 🤔

Reply:
I added this code style because current scripts are formatted in that way 😅